themes/bootstrap3/templates/RecordDriver/DefaultRecord/toolbar.phtml

```diff
@@ -17,8 +17,8 @@
     <?php $exportFormats = $this->export()->getFormatsForRecord($this->driver); ?>
     <?php if(count($exportFormats) > 0): ?>
       <li class="dropdown">
-        <a class="export-toggle toolbar-btn btn-type-export dropdown-toggle" data-toggle="dropdown" href="<?=$this->recordLink()->getActionUrl($this->driver, 'Export')?>" rel="nofollow"><?=$this->transEsc('Export Record') ?></a>
-        <ul class="dropdown-menu" role="menu">
+        <a class="export-toggle toolbar-btn btn-type-export dropdown-toggle" data-toggle="dropdown" href="<?=$this->recordLink()->getActionUrl($this->driver, 'Export')?>" rel="nofollow" aria-haspopup="true" id="export-record"><?=$this->transEsc('Export Record') ?></a>
+        <ul class="dropdown-menu" aria-owns="export-record">
```

Comment:
Is this aria-owns right? Do we really want to put the link above inside this `ul`? Maybe I'm just not understanding what is supposed to be going on here...

Reply:
I've done more research since then and this is incorrect. This should be a "menu button", as described here: https://www.w3.org/TR/wai-aria-practices-1.1/#menubutton.

That's a new big change: anchors should be for navigation, buttons for interactions.